about.html

```diff
@@ -4,70 +4,89 @@
 <head>
   <meta charset="utf-8">
 
-  <title>Sprint Challenge - About</title>
+  <title>Sprint Challenge - Home</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="about.css">
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+<header>
+  <section class=topbanner>
+    <div><img src="https://raw.githubusercontent.com/christian932/web-Sprint-Challenge-User-Interface-and-Git/master/img/lambda-black.png" alt=""></div>
+<li><a href="index.html">Home</a></li>
+<li><a href="about.html">About</a></li>
+    <li><a>Products</a></li>
+    <li><a>Blog</a></li>
+    <li><a>Contact</a></li>
     
-    How We Work
+  </section>
+  
```

Comment:
You always want to try to fill out your alt attributes to make your site more accessible.